Content.Server/Silicons/Borgs/BorgSystem.Transponder.cs

```diff
@@ -91,7 +116,12 @@ private void Destroy(Entity<ExplosiveComponent?> ent)
             return;
         }
 
-        _explosion.TriggerExplosive(ent, ent.Comp, delete: false);
+        var message = Loc.GetString(ent.Comp.DestroyingPopup, ("name", Name(ent)));
+        Popup.PopupEntity(message, ent);
+        _trigger.StartTimer(ent.Owner, user: null);
+
+        // prevent a shitter borg running into people
+        EnsureComp<AdminFrozenComponent>(ent);
```

Comment:
Prob better to just add GhostOnMove/manually make them ghost/suicide/just kill em/etc/etc since this isnt really a component made for general uses (it prevents you from suiciding for example)

Reply:
ghosting shouldnt matter if you go bye bye in 10 seconds

Reply:
What im getting at is that AdminFrozenComponent is a specialized component with a specific use-case, and not intended to be used like this. We have other general ways/components that are fine to use so we should use those

Reply:
removing InputMover instead now